### ISSUE TITLE
[Feature Request] Logarithmic Volume Control

### ISSUE THREAD

**Issue by Nixxen** [th-ch/youtube-music#1132]
Hi there,

I'm raising an issue I'm currently experiencing while using Youtube Music (and, in turn, this wrapper project since it also uses the YTM base). When I'm in voice chat with friends or colleagues and listening to music simultaneously, the music tends to overpower the voice chat, making it difficult to hear what others are saying - Even on the lowest music volume setting.

Adding a _logarithmic volume control plugin_ to the software could greatly improve this issue. Currently, YTM uses a linear volume control. [As Dr. Lex's informative write-up explains](https://www.dr-lex.be/info-stuff/volumecontrols.html), our ears do not perceive sound on a linear scale but on a logarithmic scale. This means a logarithmic volume control would align better with how our ears naturally perceive sound. This would allow users to adjust the volume of the music in much greater detail when at lower volumes and still be able to hear the voice chat, creating a more balanced listening experience.

In short: This will enable more resolution and control of music volume at lower volumes.
Thank you for considering this suggestion. Adding a logarithmic volume control plugin could improve the user experience for myself and others in similar situations.

**Comment by QuditWolf** [comment]
Enable the "exponential-volume" plugin on the ytm desktop app, I believe it does precisely that.
You might also be interested in the "precise volume" plugin.

for the website, https://greasyfork.org/en/scripts/397686-youtube-music-fix-volume-ratio/ might be of use.

https://github.com/th-ch/youtube-music#available-plugins for the plugin list.
